pkg/ui/app/containers/nodes.tsx

```diff
@@ -1,40 +1,50 @@
 import _ from "lodash";
 import * as React from "react";
-import { IInjectedProps } from "react-router";
+import { IRouter, IInjectedProps } from "react-router";
 import { connect } from "react-redux";
 
 import { refreshNodes } from "../redux/apiReducers";
 import Selector, { SelectorOption } from "../components/selector";
 import { AdminUIState } from "../redux/state";
 
+import {
+  nodeIDAttr, dashboardNameAttr,
+} from "../util/constants";
+
 import TimeScaleSelector from "./timescale";
-import { dashQueryString, nodeQueryString } from "../util/constants";
 
 interface ClusterOverviewOwnProps {
   nodes: Proto2TypeScript.cockroach.server.status.NodeStatus[];
   refreshNodes: typeof refreshNodes;
 }
 
-class ClusterOverviewState {
-  nodeOptions?: SelectorOption[] = [];
-  selectedNodes?: string[] = null;
-  selectedDash?: string;
-}
-
 type ClusterOverviewProps = ClusterOverviewOwnProps & IInjectedProps;
 
+interface ClusterOverviewState {
+  nodeOptions: { value: string, label: string }[];
+}
+
 let dashboards = [
-  { value: "node.activity", label: "Activity" },
-  { value: "node.queries", label: "SQL Queries" },
-  { value: "node.resources", label: "System Resources" },
-  { value: "node.internals", label: "Advanced Internals" },
+  { value: "activity", label: "Activity" },
+  { value: "queries", label: "SQL Queries" },
+  { value: "resources", label: "System Resources" },
+  { value: "internals", label: "Advanced Internals" },
 ];
 
 /**
  * Renders the layout of the nodes page.
  */
 class ClusterOverview extends React.Component<ClusterOverviewProps, ClusterOverviewState> {
-  state = new ClusterOverviewState();
+  // Magic to add react router to the context.
```

Comment:
I know this is from my commit - I'm curious if you've looked into it yourself at all.